ngx_pq_module.c

```diff
@@ -1528,6 +1540,25 @@ static char *ngx_pq_query_ups_conf(ngx_conf_t *cf, ngx_command_t *cmd, void *con
     ngx_pq_srv_conf_t *pscf = conf;
     return ngx_pq_prepare_query_loc_ups_conf(cf, cmd, &pscf->queries);
 }
+static char *ngx_pq_no_data_found_status_conf(ngx_conf_t *cf, ngx_command_t *cmd, void *conf) {
+    ngx_pq_loc_conf_t *p = conf;
+    ngx_int_t        *np;
+    ngx_str_t        *value;
+
+    np = (ngx_int_t *) (p + cmd->offset);
+    value = cf->args->elts;
+    *np = ngx_atoi(value[1].data, value[1].len);
+    if (*np == NGX_ERROR) {
+        return "invalid number";
+    }
+
+    if (*np != NGX_HTTP_OK && *np != NGX_HTTP_NO_CONTENT && *np != NGX_HTTP_BAD_REQUEST && *np != NGX_HTTP_UNAUTHORIZED && *np != NGX_HTTP_FORBIDDEN && *np != NGX_HTTP_NOT_FOUND)
+        return "value must be \"200\", \"204\", \"400\", \"401\", \"403\" or \"404\"";
+
+    p->no_data_found_status = *np;
+
+    return NGX_CONF_OK;
+}
```

Comment:
why not simply use [example](https://github.com/RekGRpth/ngx_http_mustach_module/blob/5e5297d213000669119daf93d458a9628093c0bf/ngx_http_mustach_module.c#L207)?